src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -111,6 +111,17 @@ class MockClientStats {
 
   std::atomic<size_t> num_active_infer_calls{0};
 
+  std::atomic<size_t> num_unregister_all_shared_memory_calls{0};
```

Comment:
Would be nice to wrap these in a struct - similar to the sequence stats - and still have them live inside of mock client stats. Partly because it is easier to confirm you are testing everything, and partly because it would be easier to compare them. You can create an expected struct, and then compare the structs.

Reply:
done